TaskHouseApi/Controllers/UsersController.cs

```diff
@@ -27,5 +27,19 @@ public IActionResult Get(int Id)
 
             return new ObjectResult(u); // 200 ok
         }
+
+        [HttpGet("{id}/location")]
+        public IActionResult GetLocation(int Id)
+        {
+            User u = unitOfWork.Users.Retrieve(Id);
```

Comment:
it dosn't need a awiat or a unitofwork.save or somethig?
- just discard this if im worng :*

Reply:
No need for await, works without. 👍